hackathon_site/dashboard/frontend/src/components/orders/OrdersFilter/OrderFilter.tsx

```diff
@@ -0,0 +1,168 @@
+    options,
+}: FieldProps & { options: StatusCategories }) => (
+    <FormGroup {...field}>
+        {options.map((item, i) => (
+            <div className={styles.filterCategory} key={i}>
+                <FormControlLabel
+                    name={field.name}
+                    value={item.status}
+                    control={<Checkbox color="primary" />}
+                    label={item.status}
+                    checked={field.value ? field.value.includes(item.status) : false}
+                />
+                <Chip
+                    size="small"
+                    label={item.numOrders}
+                    className={styles.filterCategoryChip}
+                />
+            </div>
+        ))}
+    </FormGroup>
+);
+
+const OrderFilter = ({ handleReset, handleSubmit }: FormikValues) => {
+    return (
+        <div className={styles.filter}>
+            <Paper elevation={2} className={styles.filterPaper} square={true}>
+                <form onReset={handleReset} onSubmit={handleSubmit}>
+                    <fieldset>
+                        <legend>
+                            <Typography variant="h2">Order by</Typography>
+                        </legend>
+                        <Field
+                            name="ordering"
+                            component={RadioOrderBy}
+                            options={orderByOptions}
+                        />
+                    </fieldset>
+                    <Divider
+                        className={styles.filterDivider}
+                        data-testid="orderFilterDivider"
+                    />
+                    <fieldset>
+                        <legend>
+                            <Typography variant="h2">Status</Typography>
+                        </legend>
+                        <Field
+                            name="status"
+                            component={CheckboxCategory}
+                            options={status}
+                        />
+                    </fieldset>
+                </form>
+            </Paper>
+            <div className={styles.filterBtns}>
+                <Button
+                    type="reset"
+                    color="secondary"
+                    onClick={handleReset}
+                    data-testid="clear-button"
+                >
+                    Clear all
+                </Button>
+                <Button
+                    type="submit"
+                    onClick={handleSubmit}
+                    color="primary"
+                    variant="contained"
+                    fullWidth={true}
+                    className={styles.filterBtnsApply}
+                    disableElevation
+                    data-testid="apply-button"
+                >
+                    Apply
+                </Button>
+            </div>
+        </div>
+    );
+};
+
+export const EnhancedOrderFilter = () => {
+    const onSubmit = ({ ordering, status }: OrdersFilterValues) => {
```

Comment:
Missed this last time, but we usually name functions handleSubmit, handleReset etc. the onSubmit naming convention is usually for props